packages/gatsby-remark-images/src/__tests__/index.js

```diff
@@ -369,4 +369,19 @@ describe(`showCaptions`, () => {
     expect($(`figcaption`).text()).toEqual(`some alt`)
     expect(node.value).toMatchSnapshot()
   })
+
+  it(`display nothing as caption if no title or alt`, async () => {
+    const imagePath = `images/my-image.jpeg`
+    const content = `![](./${imagePath})`
+
+    const nodes = await plugin(createPluginOptions(content, imagePath), {
+      showCaptions: true,
+    })
+    expect(nodes.length).toBe(1)
+
+    const node = nodes.pop()
+    const $ = cheerio.load(node.value)
+    expect($(`figcaption`).length).toBe(0)
+    expect(node.value).toMatchSnapshot()
```

Comment:
I'd just remove this! We're already testing what we want with the figcaption.length line 👍

Reply:
(Also great unit test 🚀 Nicely done!)

Reply:
@DSchau I assume you mean line 385? It's all marked 😆

Reply:
Not sure what "it's all marked" means--but yeah, line 385!

```suggestion
```

Reply:
@DSchau Great! I just mean that it's a comment for the whole block of code, but you're saying "I'd remove this" :-P